### DISCUSSION TITLE
How get token?

### DISCUSSION THREAD

**Discussion by lq112q** [xd2dd/dnevniklib#2] [Q&A] [answered]
Hello, how to get token?

**Comment by xd2dd** [comment, marked as answer]
https://github.com/dirtyhornet277/dnevnik_mes_token_getter

**Reply by OniKrisNeptune** [reply]
ссылка не работает

**Reply by xd2dd** [reply]
> ссылка не работает

ой, извините, чистил репозиторий, в либе появился класс для этого, почитайте в документации или напишите в ТГ dirtyhornet277

**Comment by qirieshkaclwn** [comment]


**Reply by xd2dd** [reply]
Hello, I removed this functionality due to the very large number of crutches. Get the token and Cookies
